### DISCUSSION TITLE
[@types/chart.js] ChartDataSets is missing `borderRadius`

### DISCUSSION THREAD

**Discussion by tomdye** [DefinitelyTyped/DefinitelyTyped#60304] [Issues with a @types package] [answered]
The `ChartDataSets` interface for  [@types/chart.js] appears to be missing `borderRadius` as per this example: https://www.chartjs.org/docs/3.4.1/samples/bar/border-radius.html

[a comment by an automated account is not shown]

**Reply by lekoaf** [reply]
v3 of chart.js is written in typescript. These type definitions only go up to 2.9.x.

**Reply by tomdye** [reply]
ah ha, that would explain it. Thanks. Had a PR ready and everything, was `borderRadius` not available for v2.9?

**Reply by danmana** [reply]
Nope, it was not available on v2.9.
I added the borderRadius in https://github.com/chartjs/Chart.js/releases/tag/v3.0.0-beta.5 ([see pr](https://github.com/chartjs/Chart.js/pull/7951))

**Reply by tomdye** [reply]
Thanks everyone.
